### PR TITLE
[Merged by Bors] - Add timestamp to `fluvio consume --format`

### PR DESCRIPTION
**Pull request by morenol**
This PRs adds `{{time}}` to --format that will display the timestamp with RFC3339 format, and NA in case of no timestamp in record

``` fluvio consume topic --format "{{timestamp}}"
2022-05-03T19:12:56.980Z
2022-05-03T19:13:00.481Z
```

### CODE REVIEW

File: crates/fluvio-cli/src/consume/mod.rs [hunk 1]
```diff
@@ -584,11 +588,26 @@ impl ConsumeOpt {
...
+                        )
+                    )
+                };
+
                 let object = serde_json::json!({
                     "key": formatted_key,
                     "value": value,
                     "offset": record.offset(),
                     "partition": record.partition(),
+                    "timestamp": timestamp_rfc3339,
```
Comment (sehz) [inline crates/fluvio-cli/src/consume/mod.rs:610]: how about `time` rather than `timestamp`?

**Review by sehz — CHANGES_REQUESTED** [review]

File: crates/fluvio-cli/src/consume/mod.rs (hunk 1 above)
Comment (morenol) [reply crates/fluvio-cli/src/consume/mod.rs:610]: I am ok with both options

Comment (sehz) [reply crates/fluvio-cli/src/consume/mod.rs:610]: Yes, its less typing 😉

**Review by sehz — APPROVED** [review]
LGTM

**Pull request closed without merging**